stubs/requests/requests/sessions.pyi

```diff
@@ -72,7 +72,7 @@ _Verify: TypeAlias = bool | str
 
 class Session(SessionRedirectMixin):
     __attrs__: Any
-    headers: CaseInsensitiveDict[str]
+    headers: CaseInsensitiveDict[str | None]
```

Comment:
As mentioned in https://docs.python-requests.org/en/latest/user/quickstart/#custom-headers , header values can also be bytestring.

```suggestion
    headers: CaseInsensitiveDict[str | bytes | None]
```

Reply:
Interesting. I think I added None to the wrong place. We want to allow str or bytes on actual headers, but None only on updates